docs/rfcs/015-how_we_work/README.md

```diff
@@ -0,0 +1,21 @@
+# RFC 015: How we work
+
+**Last updated: 04 September 2019.**
+
+## Background
+
+In order to work together effectively it is useful to have some guiding principals that underpin work.
+
+This is intended to be a living document and will not represent _everything all the time_.
+
+## Proposed solution
+
+### Areas
+
```

Comment:
Principles?
Things like transparency, pragmatism, collaboration etc?

Reply:
Yes. 

This isn't a complete list! Please PR things you want to see.

Reply:
Isn't this a PR?

Reply:
Either works - I'd prefer to merge this and then add things.